src/components/editArrangementTitle/editArrangementTitle.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { Typography, TextField } from '@material-ui/core'
+
+export class EditArrangementTitle extends Component {
+    constructor (props) {
+        super(props)
+
+        this.handleKeyPress = this.handleKeyPress.bind(this)
+        this.escFunction = this.escFunction.bind(this);
+    }
+    
+    state = {
+        name: ''
```

Comment:
I think changing this to this.props.name fixes the default value thing you said.

Reply:
agreed

Reply:
Thank you! I was trying to use this.props.real.name, but was getting a NULL error. Guess I gotta study redux more :)